PrivacyPolicy.md

```diff
@@ -0,0 +1,86 @@
+
+The safety and security of your information also depends on you. For example, with respect to your nelson.social account, you are responsible for keeping your account name and password confidential. We ask you not to share your password with anyone.
+
+Unfortunately, the transmission of information via the Internet is not completely secure. Although we do our best to protect your personal information, we cannot guarantee the security of your personal information transmitted to the nelson.social online space. Any transmission of personal information is at your own risk. We are not responsible for circumvention of any privacy settings or security measures contained on the nelson.social online space.
+
+## Data Retention 
+
+Except as otherwise permitted or required by applicable law or regulation, we will only retain your personal information for as long as necessary to fulfill the purposes we collected it for , including for the purposes of satisfying any legal, accounting, or reporting requirements. Under some circumstances we may anonymize your personal information so that it can no longer be associated with you. We reserve the right to use such anonymous and de-identified data for any legitimate business purpose without further notice to you or your consent.
+
+## Accessing and Correcting Your Personal Information 
+
+It is important that the personal information we hold about you is accurate and current. Please keep us informed if your personal information changes. By law, you have the right to request access to and to correct the personal information that we hold about you.
+
+We may request specific information from you to help us confirm your identity and your right to access, and to provide you with the personal information that we hold about you or make your requested changes. Applicable law may allow or require us to refuse to provide you with access to some or all of the personal information that we hold about you, or we may have destroyed, erased, or made your personal information anonymous in accordance with our record retention obligations and practices. If we cannot provide you with access to your personal information, we will inform you of the reasons why, subject to any legal or regulatory restrictions.
+
+If you are concerned about our response or would like to correct the information provided, you may contact `admin@nelson.social`.
+
+## Changes to Our Privacy Policy 
+
+It is our policy to post any changes we make to our privacy policy on this page. We include the date the privacy policy was last revised at the top of the page. You are responsible for ensuring we have an up-to-date, active, and deliverable email address for you, and for periodically this privacy policy to check for any changes.
+
+## Contact Information and Challenging Compliance
+
+We welcome your questions, comments, and requests regarding this privacy policy and our privacy practices. Please contact `admin@nelson.social`.
+
```

Comment:
Can we attribute this document? Is it copied from or inspired by another one that we can reference? Can we put a license on it so others could use it?

Reply:
Agreed, do you want to add a section to this PR?